### PR TITLE
Tadpole balance: medbus edition

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Man, tadpole hasn't been touch in a while.

Give tadpole a dedicated wall dispenser that can refill specific autoinjectors.

Give tadpole fire extinguisher wall dispenser because fire bad.

## Why It's Good For The Game

When the share NanoMed Plus got merged, it nerfed medbus. This is to ensure that medbus' balance is relevant and not have to go loot planetside for a NanoMed Plus.

Also, this will buff tadpole.

## Changelog
:cl:
balance: tadpole gets dedicated wall dispenser that can refill specific autoinjectors
/:cl:

### CODE REVIEW

**Comment by Pill-U** [comment]
You could also just bring a medical vendor and a fire extinguisher on tadpole manually. I like getting creative with that sort of stuff and making tad into what you like.

**Comment by Losenis** [comment]
>You could also just bring a medical vendor

Vendors are no unwrenchable and shared

**Comment by FosterGrime-Heymynameslouie** [comment]
what will be the contents of the dedicated closets though

File: code/game/objects/machinery/vending/vending_types.dm [hunk 1]
```diff
@@ -421,6 +421,23 @@
...
+		/obj/item/reagent_containers/hypospray/autoinjector/kelotane = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/tricordrazine = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/tramadol = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/hypervene = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 1,
+		/obj/item/stack/medical/heal_pack/gauze = 2,
+		/obj/item/stack/medical/heal_pack/ointment = 2,
+		/obj/item/healthanalyzer = 1,
+		/obj/item/stack/medical/splint = 1,
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/vending_types.dm:439]: If you want it to be refill set them to 0

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/vending/vending_types.dm [hunk 2]
```diff
@@ -421,6 +421,23 @@
...
+		/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/tricordrazine = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/tramadol = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/hypervene = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 0,
+		/obj/item/stack/medical/heal_pack/gauze = 2,
+		/obj/item/stack/medical/heal_pack/ointment = 2,
+		/obj/item/healthanalyzer = 1,
+		/obj/item/stack/medical/splint = 1,
+	)
```
Comment (BraveMole) [inline code/game/objects/machinery/vending/vending_types.dm:440]: are you sure you only want to authorize this? What about pill bottle? This should be able to hold as much as hold vendor imo

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/vending/vending_types.dm (hunk 2 above)
Comment (novaepee) [reply code/game/objects/machinery/vending/vending_types.dm:440]: The previous NanotrasenMed Plus only hold peridaxon and Russian Red.

While it's true that NanotrasenMed Plus hold med rig and by extension pill bottles, it's too far of a stretch to claim that NanotrasenMed Plus holds pill bottles like a MarineMed.

**Comment by Losenis** [comment]
Tried it, found out it has no combat injectors. Marines had taken all of them too so no bringing over a bunch. Rest of the items there even if useful will run out extremely fucking fast because they're only 1~2 stacks and not the refillable kind. Effectively, no usage and doesn't even begin to compensate for the inability to bring a vendor over when some maps have medbay parts underground so you can't even nab one from there.

image

Consider giving it the reagent bottles from the medvendors instead of items if you feel it'll be too big of a buff for normal unprepared marines.

**Pull request merged**